chore: Remove deploy page from peerjs guide (#40155)
mdn/content: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/end_a_call/index.md

> [!WARNING]
> The way we've currently coded things means that when a connection is closed, both browsers will be updated **only** if the person who started the call presses "Hang up" first. If the person who answered the call clicks "Hang up" first, the other caller will also have to click "Hang up" to see the correct HTML.

Now we're finished with the project!

> The next step you could take is to [deploy it to a hosting provider that supports Node.js](/en-US/docs/Learn_web_development/Extensions/Server-side/Express_Nodejs/deployment) projects.
> 
> ## Further Reading
> 
> Here's some useful resources related to this guide:

Reviewer: ```suggestion
Next, you could [deploy it to a hosting provider that supports Node.js projects](/en-US/docs/Learn_web_development/Extensions/Server-side/Express_Nodejs/deployment).

## See also
```

- [PeerJS](https://peerjs.com/)
- [WebRTC](/en-US/docs/Web/API/WebRTC_API)
- [PeerJS Server](https://github.com/peers/peerjs-server)
- [A similar video tutorial with video](https://www.youtube.com/watch?v=OOrBcpwelPY)
- [The code tutorial](https://github.com/SamsungInternet/WebPhone/tree/master/tutorial)

> {{Previous("Web/API/WebRTC_API/Build_a_phone_with_peerjs/Connect_peers/Answer_a_call")}}

Author: Although I don't see any visible difference:

![image](https://github.com/user-attachments/assets/f9d153e3-cf62-4adc-a8ea-234bad01500e)
